### PR TITLE
Fix README

### PR DESCRIPTION
**Pull request by arnavs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,113 +10,19 @@ Welcome to our Mozilla Study Group! A few things to do & know now that you're he
...
+ - **Add yourself to the website:** If you'd like to appear on the website under the 'Who we Are' section, have a look at the `_data/members.yml`; send us a pull request with an entry for yourself, or open an issue and we'll do it for you. Basically, we need the following information:
 
-### Optional: Setting up a Google Calendar for your Study Group
-
-If you'd like to offer your community a calendar of events they can import into their own calendars, try using a Google Calendar. To set up, make a new google account, and update the variables in `_config.yml` under the heading 'Setup Google Calendar'.
-
-You can add events to your calendar by hand, but if you'd like to manage it automatically, there's a script to do so in `scripts/updateCalendar.py`; instructions for use are at the top of that file.
-
-## How to Launch a New Event
```
Comment (ttimbers) [inline README.md:64]: Can we keep this in the README or move it to another file @arnavs ? I use it as a reference when I forget how to do this...

Comment (arnavs) [reply README.md:64]: My thought was that when we needed to do this, we can just check the file history. But whatever you prefer.

Comment (arnavs) [reply README.md:64]: Making `admin.md` probably isn't the worst idea. Then we could also put in some UBC-specific tips (i.e., if you need a room, talk to so-and-so.)

Comment (arnavs) [reply README.md:64]: @ttimbers Done.

**Pull request merged**